web-client/src/presenter/computeds/messageDocumentHelper.js

```diff
@@ -99,6 +108,11 @@ export const messageDocumentHelper = (get, applicationContext) => {
   const showServePetitionButton =
     showNotServed && isPetitionDocument && permissions.SERVE_PETITION;
 
+  const showSignStipulatedDecisionButton =
+    isInternalUser &&
+    caseDocument.eventCode === PROPOSED_STIPULATED_DECISION_EVENT_CODE &&
```

Comment:
And I think we already have this with `isStipulatedDecision`

Reply:
This is `PROPOSED_STIPULATED_DECISION_EVENT_CODE` vs `STIPULATED_DECISION_EVENT_CODE`.